docs/tutorial.rst

```diff
@@ -338,7 +340,7 @@ We could do that simply by setting the :attr:`valid_prog_environs` as follows:
 
    self.valid_prog_environs = ['+stream']
 
-This tells ReFrame that this test is valid only for environments that define the ``stream`` feature.
+This tells ReFrame that this test is valid only for environments that define the ``stream`` feature; the ``+`` prefix requests a feature. The syntax defined in :attr:`valid_prog_environs` allows for other requirements.
```

Comment:
```suggestion
This tells ReFrame that this test is valid only for environments that define the ``stream`` feature (the ``+`` prefix requests a feature; we will explain the syntax later in this section).
```

And then expand later in the bullet list that talks about features and extras (I can do that).